testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/cli/UndeployTestCase.java

```diff
@@ -114,10 +134,22 @@ public void testLegacyUndeployWithAllServerGroups() throws Exception {
         checkExist(result, cliTestApp1War.getName(), ENABLED, ctx);
 
         // From serverGroup1 only, still referenced from sg2. Must keep-content
-        ctx.handle("undeploy --server-groups=" + sgOne + ' ' + cliTestApp1War.getName());
+        try {
+            ctx.handle("undeploy --server-groups=" + sgOne + ' ' + cliTestApp1War.getName());
+        } catch (CommandLineException ex) {
+            // Check error message
+            assertThat("Error message doesn't contains expected message information!",
+                    ex.getMessage(),
+                    allOf(containsString("Cannot undeploy"),
+                            containsString(cliTestApp1War.getName()),
+                            containsString(" as it is still deployed in "),
```

Comment:
It seems that this PR contains more fixes than is is described in https://issues.jboss.org/browse/WFCORE-3566

Can you please update description of WFCORE-3566 (and the description of this PR) and add information of all changes from this PR?

Reply:
@marekkopecky thanks for the review, I will eventually update the description in JIRA